IronFoundry.Container.Shared/IContainerDirectory.cs

```diff
@@ -55,7 +55,7 @@ public static ContainerDirectory Create(FileSystemManager fileSystem, string con
             var containerUserPath = Path.Combine(containerPath, UserRelativePath);
             var containerBinPath = Path.Combine(containerPath, BinRelativePath);
 
-            fileSystem.CreateDirectory(containerPath, GetContainerDefaultAccess());
+            fileSystem.CreateDirectory(containerPath, GetContainerUserAccess(containerUser.UserName, FileAccess.Read));
```

Comment:
My only concern here is that we are storing some files in the root of the container (specifically the Container properties.json, see this line of code: https://github.com/cloudfoundry-incubator/if_warden/blob/extractContainerLib/IronFoundry.Container/ContainerService.cs#L66). 

If we are going to allow read access to files at the root, then I want to move any "system" files into another location.  I was thinking of adding another root-level path to the container, maybe `sys`, `etc` (for configuration data), or `private`.

I would prefer that we move the properties.json file to another sub-folder when we make this change.